### PR TITLE
[RM Ch03] Add redfish as Hardware Infrastructure Manager option

### PR DESCRIPTION
**Pull request by Bharath-KKB**
Closed the old PR that included ODIM. This new PR proposes redfish as an option for the Hardware Infrastructure Manager in chapter 3 of the reference model.

### CODE REVIEW

**Review by collivier — CHANGES_REQUESTED** [review]
Because of RTD forcing the use of .readthedocs.yaml, we don't have another choice than splitting the git repository per stream. This PR must be applied on next RM's git repository and not here.

Everything is already ready on my own repositories but sadly I don't have the rights needed on anuket-project organization. **Please give me them to complete the migration.**

Be free to ask for any  detail about the migration during a Weekly Technical meeting.

**Review by walterkozlowski — APPROVED** [review]

**Review by joaoarodrigues — COMMENTED** [review]
TEST

File: doc/ref_model/chapters/chapter03.rst [hunk 1]
```diff
@@ -357,10 +357,53 @@ The hardware infrastructure manager needs to support the following functional as
 - **Additional Management Functions**: include software and configuration life cycle management, identity management,
   access management, policy management (e.g. to enforce security policies), etc.
 
-Left for future use
--------------------
+The Redfish® Standard based Hardware Infrastructure Manager 
+-----------------------------------------------------------
 
-This section is left blank for future use
+This section proposes a Redfish based hardware infrastructure manager for the Anuket project. ODIM software architecture is an example of an open source project implementing Redfish standard.
```
Comment (petorre) [inline doc/ref_model/chapters/chapter03.rst:363]:
```suggestion
This section proposes a Redfish based hardware infrastructure manager for the Anuket project. Open Distributed Infrastructure Management (ODIM) is an open source software platform that delivers means of distributed physical infrastructure lifecycle management based on the industry standard DMTF Redfish API and Model specification.
```

File: doc/ref_model/chapters/chapter03.rst [hunk 2]
```diff
@@ -357,10 +357,53 @@ The hardware infrastructure manager needs to support the following functional as
...
 
-Left for future use
--------------------
+The Redfish® Standard based Hardware Infrastructure Manager 
+-----------------------------------------------------------
 
-This section is left blank for future use
+This section proposes a Redfish based hardware infrastructure manager for the Anuket project. ODIM software architecture is an example of an open source project implementing Redfish standard. 
+
+Chapter 9 (:ref:`chapters/chapter09:Configuration and Lifecycle Management`) of this reference model specifies redfish standard for managing Infrastructure Hardware. The GSMA's "Cloud Infrastructure Reference Model (NG.126)" also specifies Redfish as the standard interface that should be
```
Comment (petorre) [inline doc/ref_model/chapters/chapter03.rst:365]:
```suggestion
Chapter 9 (:ref:`chapters/chapter09:Configuration and Lifecycle Management`) of this reference model specifies Redfish standard for managing Infrastructure Hardware. The GSMA's "Cloud Infrastructure Reference Model (NG.126)" also specifies Redfish as the standard interface that should be 
```

File: doc/ref_model/chapters/chapter03.rst [hunk 3]
```diff
@@ -357,10 +357,53 @@ The hardware infrastructure manager needs to support the following functional as
...
+
+-	**Resource Allocation and Composition**: The Redfish schema has a composition model through which a client can compose a logical resource by allocating physical resources. `Redfish Composition Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2050_1.2.0_1.pdf>`__
+
+-	**Underlay Network Resources Manager**: The Redfish schema has models for fabrics, switches and SmartNICs.
+
+-	**Monitoring**: The Redfish schema contains event model for the client to receive hardware events and telemetry model for collecting information across the entire model (physical and functional) `Redfish Telemetry Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2051_1.0.0.pdf>`__
+
+-	**Additional Management Functions**: The Redfish schema has models for access management and identity management. Redfish has published work-in-progress extension for support policy management and OS/VM management `Policy Model Proposal, Redfish for Operating Systems, Applications and Containers <https://www.dmtf.org/documents/redfish-spmf/redfish-operating-systems-applications-and-containers-05wip>`__
+
+How redfish fits into the ETSI NFV Networking Reference Model
```
Comment (petorre) [inline doc/ref_model/chapters/chapter03.rst:397]:
```suggestion
How Redfish fits into the ETSI NFV Networking Reference Model
```

**Comment by walterkozlowski** [comment]
@collivier, is there a way we can merge this, and discuss the repo issues somewhere else?

**Review by joaoarodrigues — APPROVED** [review]

File: doc/ref_model/chapters/chapter03.rst [hunk 4]
```diff
@@ -357,10 +357,53 @@ The hardware infrastructure manager needs to support the following functional as
...
+
+-	**Resource Allocation and Composition**: The Redfish schema has a composition model through which a client can compose a logical resource by allocating physical resources. `Redfish Composition Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2050_1.2.0_1.pdf>`__
+
+-	**Underlay Network Resources Manager**: The Redfish schema has models for fabrics, switches and SmartNICs.
+
+-	**Monitoring**: The Redfish schema contains event model for the client to receive hardware events and telemetry model for collecting information across the entire model (physical and functional) `Redfish Telemetry Whitepaper <https://www.dmtf.org/sites/default/files/standards/documents/DSP2051_1.0.0.pdf>`__
+
+-	**Additional Management Functions**: The Redfish schema has models for access management and identity management. Redfish has published work-in-progress extension for support policy management and OS/VM management `Policy Model Proposal, Redfish for Operating Systems, Applications and Containers <https://www.dmtf.org/documents/redfish-spmf/redfish-operating-systems-applications-and-containers-05wip>`__
```
Comment (collivier) [inline doc/ref_model/chapters/chapter03.rst:395]: this link isn't publicly readable (access - denied).
Would you mind replacing it or worse making an linkcheck exception in setup.py

**Comment by collivier** [comment]
@walterkozlowski be free to merge it if urgent but I would rather suggest to fix the link which is access denied.
No worries I will manage the cherry-picking in the new repos.

**Review by walterkozlowski — APPROVED** [review]

**Review by collivier — APPROVED** [review]

**Comment by collivier** [comment]
@walterkozlowski I merge it but the current build is broken because of wrong link in chapter 7 (out this scope)
https://www.cyber.gov.au/acsc/view-all-content/ism/ -> https://www.cyber.gov.au/resources-business-and-government/essential-cyber-security/ism ?

**Pull request merged**